tools/src/main/scala/caliban/tools/IntrospectionClient.scala

```diff
@@ -12,8 +12,7 @@ import caliban.parsing.adt.Definition.TypeSystemDefinition.{ DirectiveDefinition
 import caliban.parsing.adt.Type.{ ListType, NamedType }
 import caliban.parsing.adt.{ Directive, Document, Type }
 import sttp.client3._
-import sttp.client3.asynchttpclient.zio.AsyncHttpClientZioBackend
-import sttp.client3.asynchttpclient.zio._
+import sttp.client3.asynchttpclient.zio.{ AsyncHttpClientZioBackend, _ }
```

Comment:
We can simply remove `AsyncHttpClientZioBackend` I think?

Reply:
No, it is used on 26 line

Reply:
But since we import `_`, it's not needed. Right?

Reply:
No, it's used for 29-30 lines. If it was unused, then after optimizing imports it should fade away

Reply:
What I mean is that it's already imported by `_` so there is no need to specify it. Just try to remove it and only keep `_`, you'll see.

Reply:
Sorry, doesn't noticed that)